pep-9999.rst

```diff
@@ -0,0 +1,892 @@
+PEP: 9999
+Title: Metadata for Python Software Packages 2.2
+Version: $Revision$
+Last-Modified: $Date$
+Author: Philippe Ombredanne <pombredanne at nexb.com>
+Sponsor: Paul Moore <p.f.moore at gmail.com>
+BDFL-Delegate: Paul Moore <p.f.moore at gmail.com>
+Discussions-To: https://discuss.python.org/t/improving-license-clarity-with-better-package-metadata
```

Comment:
I prefer using the ID here since the slug changes if you edit the title. This should reliably point to the same thread: https://discuss.python.org/t/2154

Reply:
@uranusjr done